core/dbt/flags.py

```diff
@@ -1,7 +1,9 @@
-import os
+# Do not import the os package because we expose this package in jinja
```

Comment:
I don't understand this comment. I believe the `os` package is still running, even if only a few objects are pulled in from that namespace. In other words, I don't think this is doing anything different than the original version.